### PR TITLE
Feature/filter documents cx 1703

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
Following the conversation within [this PR](https://github.com/onfido/onfido-sdk-ui/pull/228) we are introducing the possibility of filtering document types.

# Solution
This PR allows the customisation of document types. The document step options can be customised by passing an object containing the document types as keys and boolean as values, as in the example below.
```javascript
{
    type:'document',
    options:{
        // `passport` will be the only available option on the document selection screen
        documentTypes: { passport: true }
     }
}
  ```

For some reason I can only add one reviewer. @TheComfyChair could I have your feedback on this PR, please?

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [x] Have new manual tests been written down?
- [x] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://327-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 1]
```diff
@@ -282,7 +282,8 @@ A number of options are available to allow you to customise the SDK:
 
   ### document ###
 
-  This is the document capture step. Users will be asked to select the document type and to provide images of their selected documents. They will also have a chance to check the quality of the images before confirming. No customisation options are available for this step.
+  This is the document capture step. Users will be asked to select the document type and to provide images of their selected documents. They will also have a chance to check the quality of the images before confirming. The custom options are:
+  - documentTypes (object - it can contain the following keys: `passport`, `driving_licence`, `national_identity_card`. The value for each key should be a boolean)
```
Comment (rfreitas) [inline README.md]: are these types the same of the api?

Comment (stefaniacardenas) [reply README.md]: Yes

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://327-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
Great works well. I will approve and merge

**Pull request merged**